docs/self-hosted/kubernetes-migration-guide.md

```diff
@@ -144,7 +144,9 @@ After each scale command, use `kubectl get pods` again to verify that the redis
 ```
 export PV_NAME=$(kubectl get pvc redis-data-redis-0 -ojsonpath='{.spec.volumeName}')
 kubectl scale statefulset -n default redis --replicas 0
-rm /var/openebs/local/$PV_NAME/appendonly.aof
+# Depending on your setup, you might need to run the following three commands as `root`
+rm -f /var/openebs/local/$PV_NAME/appendonly.aof
+rm -rf /var/openebs/local/$PV_NAME/appendonlydir
```

Comment:
Just to verify, looking at the command output shared in the Trello ticket, I see these files listed (for the user that reported it):

```
-rw-r--r--. 1 root root 89 Nov 27 13:36 appendonly.aof.1.base.rdb
-rw-r--r--. 1 root root 39444361 Nov 28 10:10 appendonly.aof.1.incr.aof
-rw-r--r--. 1 root root 88 Nov 27 13:36 appendonly.aof.manifest
```

They won't be removed by these commands. I don't think it's an issue because `redis-start.sh` just checks that `/data/appendonly.aof` is absent before importing the data, but I wonder if could these pose an additional problem somehow? Should we add a wildcard in our delete commands, or can we not bother with these?

Reply:
Doh, never mind @IgorBenko , it's in that subdirectory `appendonlydir`, I didn't spot that :)